### PR TITLE
chore(server): fix ups for comments on #2945

### PR DESCRIPTION
**Pull request by elliotclee**
Sorry for the extra hassle - this PR consists solely of changes that should have been in #2945 but which I didn't push to the right place in time. It just sorts the list of raw mime types and removes a large comment that is unnecessary. Thanks to @uhthomas for noticing.

### CODE REVIEW

File: server/src/domain/domain.constant.ts [hunk 1]
```diff
@@ -67,33 +67,35 @@ const validMimeTypes = [
   'image/x-sony-arw',
   'image/x-sony-sr2',
   'image/x-sony-srf',
-  'image/dng',
+
+  'image/3fr',
```
Comment (uhthomas) [inline server/src/domain/domain.constant.ts:71]: Ah, to clarify: Could we merge all of these into one contiguous list and then sort alphabetically?

**Review by jrasm91 — APPROVED** [review]
Can we add a jest test that asserts the mime type order is sorted?

File: server/src/domain/domain.constant.ts [hunk 2]
```diff
@@ -29,15 +29,40 @@ export function assertMachineLearningEnabled() {
...
 
 const validMimeTypes = [
+  'image/3fr',
+  'image/ari',
+  'image/arw',
   'image/avif',
+  'image/cap',
+  'image/cin',
+  'image/cr2',
+  'image/cr3',
```
Comment (uhthomas) [inline server/src/domain/domain.constant.ts:39]: Duplicate

**Review by uhthomas — APPROVED** [review]

**Comment by uhthomas** [comment]
Looks great @elliotclee, thanks! As @jrasm91 suggested, a test which verifies the order of this list would be much appreciated.

**Comment by elliotclee** [comment]
> Can we add a jest test that asserts the mime type order is sorted?

I don't know how to write this type of test. If you point me to an example in immich I should be able to put together the logic OK. It will be a few days in any case - my electricity is getting disconnected tomorrow to allow for repairs.

**Comment by jrasm91** [comment]
> > Can we add a jest test that asserts the mime type order is sorted?
> 
> I don't know how to write this type of test. If you point me to an example in immich I should be able to put together the logic OK. It will be a few days in any case - my electricity is getting disconnected tomorrow to allow for repairs.

Add something like this in `asset-upload.config.spec.ts`. You will need to **export** const validMimeTypes so it can be used in the test though.

```typescript
// domain.constant.ts
export const validMimeTypes = [...]

// asset-upload.config.spec.ts
it('should be a sorted list', () => {
  expect(validMimeTypes).toEqual([...validMimeTypes].sort());
});
```

**Comment by uhthomas** [comment]
I've added the tests in #3001.

**Comment by alextran1502** [comment]
superseded by #3001

**Pull request closed without merging**